src/client.cpp

```diff
@@ -1757,8 +1757,17 @@ void Client::afterContentReceived(IrrlichtDevice *device)
 	draw_load_screen(text, device, guienv, 0, 72);
 	m_nodedef->updateAliases(m_itemdef);
 	std::string texture_path = g_settings->get("texture_path");
-	if (texture_path != "" && fs::IsDir(texture_path))
-		m_nodedef->applyTextureOverrides(texture_path + DIR_DELIM + "override.txt");
+	if (texture_path != "" && fs::IsDir(texture_path)) {
+		std::vector<std::string> paths;
+		char ignore[] = { '_', '.' };
+		std::vector<char> vector(ignore, ignore + sizeof(ignore) / sizeof(ignore[0]));
```

Comment:
could you name this variable something other than... vector..?  like maybe chars_to_ignore?